api/v1/storagecluster_types.go

```diff
@@ -210,7 +210,10 @@ type ManageCephBlockPools struct {
 
 // ManageCephNonResilientPools defines how to reconcile ceph non-resilient pools
 type ManageCephNonResilientPools struct {
-	Enable bool `json:"enable,omitempty"`
+	Enable              bool                         `json:"enable,omitempty"`
+	Count               int                          `json:"count,omitempty"`
+	Resources           corev1.ResourceRequirements  `json:"resources,omitempty"`
+	VolumeClaimTemplate corev1.PersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
```

Comment:
Add some comments about what these fields mean and how they can be used. When CRD is updated, it'll include these comments and is easier to document.